src/main/java/bio/terra/cloudfunctions/streaming/GcsBQ.java

```diff
@@ -20,6 +37,87 @@
 
 public class GcsBQ implements BackgroundFunction<GCSEvent> {
   private static final Logger logger = Logger.getLogger(GcsBQ.class.getName());
+  private static final String DATASET = "simple_stream_dataset";
+  private static final String TABLE = "SUMMARY_testRun";
+  private static final Schema SCHEMA =
```

Comment:
Should these hard-coded values live in the test runner library instead? This seems very similar to the BQ schemas defined [here](https://github.com/DataBiosphere/terra-test-runner/tree/main/bq) and the POJO defined [here](https://github.com/DataBiosphere/terra-test-runner/blob/f963972154a550796a0102d0b85c1ebe5695b4d3/src/main/java/bio/terra/testrunner/runner/TestRunner.java#L47). I'm nervous about duplicating schema definitions across multiple repos.

Reply:
The schemas in the `terra-test-runner` repo will be updated to match the schemas in the `resources` folder. Once we work out how we are going to integrate Terraform with the cloud function, it will become clear to us where these schemas should reside permanently. For now I will just create a Jira ticket for future clean up. Also see my response to UC's comments. I have refactored the GHA to provide `envvars` and replaced the hardcoded values in `GcsBQ.java` with `envvars` as well.